### PR TITLE
Adding 7-get-page template. closes #2101

### LINKED ISSUE #2101: Create 7-get-page template
Should supports subway map pattern. 

**Example pages**
- https://www.vets.gov/education/apply-for-education-benefits/
- https://www.vets.gov/disability-benefits/apply-for-benefits/

Related to #2099. Blocks #2089. Blocked by #2100

### PR DESCRIPTION
**Pull request by webinista**
modified:   _disability-benefits/apply-for-benefits.md
modified:   _education/apply-for-education-benefits.md
new file:   _includes/7-get-page.html
modified:   _layouts/page-breadcrumbs.html

Blocks #2366

### CODE REVIEW

File: _disability-benefits/apply-for-benefits.md [hunk 1]
```diff
@@ -22,30 +22,23 @@ relatedlinks:
     description: ""
 ---
 
-<div class="main interior" role="main" markdown="0">
-<div class="section one" markdown="0">
-<div class="row" markdown="0">
-<div class="small-12 columns"  markdown="1">
-
 ### Who should read this?
```
Comment (emilyville) [inline _disability-benefits/apply-for-benefits.md]: @webinista take a look at these two and compare:

https://vets-website-pr-2401.herokuapp.com/disability-benefits/apply-for-benefits/
https://www.vets.gov/disability-benefits/apply-for-benefits/

I think some of the breathing room in the design got lost in this change. (sorry this is a bit of a nit!)

**Comment by emilyville** [comment]
Take a look at the comment - let me know if you see what I'm seeing!

File: _disability-benefits/apply-for-benefits.md (hunk 1 above)
Comment (webinista) [inline _disability-benefits/apply-for-benefits.md]: Ok. We'll need to add a second, alternative template then.

**Comment by emilyville** [comment]
LGTM!

**Pull request merged**